topdown/eval.go

```diff
@@ -3258,3 +3252,12 @@ func suppressEarlyExit(err error) error {
 	}
 	return ee.prev // nil if we're done
 }
+
+func copyExprSansTerms(expr *ast.Expr) *ast.Expr {
```

Comment:
💭 I wonder if we should expose this on the ast.Expr struct itself... the reason being if we introduce other fields into ast.Expr, I could imagine missing this function and introducing a subtle bug.

Reply:
Updated.